### DISCUSSION TITLE
Week 3: FreeCodeCamp vs CSS Diner

### DISCUSSION THREAD

**Discussion by kassyoveson** [USU-ITLS-x265-23-summer/class-repo#76] [Week 3 - Discussions]
The JavaScript, HTML, and CSS sections of FreeCodeCamp are all well-designed and comprehensive, and they cover a wide range of topics. The courses are also interactive and engaging, and they provide a great way to learn the basics of web development.

**FreeCodeCamp**: I find FreeCodeCamp interactive and engaging and has community support built right into the program. I find it to be very challenging as a beginner and time consuming. I don’t always understand what I did wrong even when the give me a ‘hint’ of what I need to fix. Thank heavens for the internet!

**CSS Dine**r is a fun and interactive way to learn CSS. The game challenges you to style different elements on a page, and it provides immediate feedback on your work. CSS Diner is a great way to learn the basics of CSS, and it's also a lot of fun.
Overall, FreeCodeCamp and CSS Diner are both great resources for learning web development. FreeCodeCamp is a more comprehensive option, while CSS Diner is a more fun and interactive option. Ultimately, the best choice for you will depend on your learning style and goals.

**Comment by Skye404** [comment]
I'm right there with you on the FreeCodeCamp exercises. I wouldn't make it through without the online helps. The helps within the programs aren't very helpful, unfortunately.
